homeassistant/components/tuya/binary_sensor.py

```diff
@@ -140,9 +140,9 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
     # https://developer.tuya.com/en/docs/iot/categorycwwsq?id=Kaiuz2b6vydld
     "cwwsq": (
         TuyaBinarySensorEntityDescription(
-            key=DPCode.FEED_STATE,
-            translation_key="feeding",
-            on_value="feeding",
+            key=DPCode.CHARGE_STATE,
+            translation_key="charge_state",
+            on_value="charge_state",
```

Comment:
Why is this replacing the existing sensor?

Reply:
Could not find a sensor for charge_state. Please check https://github.com/home-assistant/core/issues/130039 for more info on the issue.